src/Eto/Forms/ThemedControls/ThemedDocumentControlHandler.cs

```diff
@@ -172,9 +173,9 @@ public Color TabBackgroundColor
 	}
 
 	/// <summary>
-	/// Gets or sets the highlight background color for the highlighted  tab.
+	/// Gets or sets the highlight background color for the highlighted (or active) tab.
 	/// </summary>
-	/// <value>The highlight background color for the close highlighted tab.</value>
+	/// <value>The highlight background color for the highlighted (or active) tab.</value>
```

Comment:
"(or active)" I think the word `active` is not used across the file but instead the word `selected` is used.
I would rephrase it as following, 
`The background color for the **selected** tab` 

This is how it is document by Microsoft documentation.
![image](https://github.com/picoe/Eto/assets/36695620/442f50e8-cb5a-4040-a2dd-e9114663c67a)